### DISCUSSION TITLE
Feature Request - Server Update Notification

### DISCUSSION THREAD

**Discussion by Daniel-MFG** [dani-garcia/vaultwarden#4420] [Ideas]
It would be really nice to have some kind of automated "New Update Availible Notification" option...
If we currently login to the "Admin Panel" we can see if there is a new Version in "Diagnostics".

It would be very handy if we could set a mail address (or use the address of the admin) to which the server sends a mail whenever it detects there is a new version. (and maybe simply check once a day if there is a new version)

This feature doesn't need to do anything fancy...
But sometimes when you got a lot going on you forget to check for new versions manually all the time and then this feature could save you a lot of time.

**Comment by BlackDex** [comment]
Use GitHub notifications for this.
Just subscribe to releases and you will receive a mail for this from GitHub

**Reply by Daniel-MFG** [reply]
I actually didn't know that this was an option.
(and for me that is gonna be good enough for now)
I would still really love to see a feature like this as i would prefer to be able to do this without needing a github account...
(especially if i were to use this in a business environment)

But thanks for telling me this GitHub feature exists!

**Reply by redalert99** [reply]
You can use watchtower to automatically update the docker container if you use docker.

**Reply by Daniel-MFG** [reply]
@redalert99 I know but i don't even want to automatically update.
(because of the potential that maybe i have to change someting else like it will be with v1.31.0 - proxy settings needing to be updated)

So i prefer to do them manually anyway
